makefile

```diff
@@ -38,6 +38,25 @@ test: ## Runs the unit test suite.
 	@make install-node-modules
 	docker compose ${DEV_COMPOSE_FILES} run --rm --no-deps ui npm run test
 
+BASE_URL ?= "http://localhost:3000"
```

Comment:
Is there a reason you don't use `CYPRESS_BASE_URL` here?

Reply:
Yes. If you have CYPRESS_BASE_URL set locally it will override the baseUrl parameter.

Reply:
OK I can live with it - might need to bear it in mind if anyone has any problems.